### ISSUE TITLE
Breaks with latest typings file

### ISSUE THREAD

**Issue by endquote** [yuanqing/create-figma-plugin#54]
Note the change here, which declares browser APIs like `console`: https://github.com/figma/plugin-typings/commit/545a2c08bae3eec60c569dd74b395f997fca5f40

This results in errors like this when running the `watch` or `build` scripts:

```
error TypeScript error

    node_modules/@figma/plugin-typings/index.d.ts:10:9 - error TS2451: Cannot redeclare block-scoped variable 'console'.

    10   const console: Console
               ~~~~~~~

      node_modules/typescript/lib/lib.dom.d.ts:17792:13
        17792 declare var console: Console;
                          ~~~~~~~
        'console' was also declared here.

error TypeScript error

    node_modules/typescript/lib/lib.dom.d.ts:17792:13 - error TS2451: Cannot redeclare block-scoped variable 'console'.

    17792 declare var console: Console;
                      ~~~~~~~

      node_modules/@figma/plugin-typings/index.d.ts:10:9
        10   const console: Console
                   ~~~~~~~
        'console' was also declared here.
```

I tried removing `dom` from the `lib` attribute in `node_modules/@create-figma-plugin/tsconfig/tsconfig.json`, but that didn't work.

**Comment by yuanqing** [comment]
@endquote Thanks for the detailed report! The temporary fix for now seems to be to use version `1.3.1` of `@figma/plugin-typings`, which doesn’t define types for `console`?

**Comment by yuanqing** [comment]
Just cut a `1.3.4` of `create-figma-plugin` that pins `@figma/plugin-typings` to the `1.31.0` version (see https://github.com/yuanqing/create-figma-plugin/commit/5c421c6fc148753378c7749f3c5588bfa60ea89e) until there’s a proper fix in place

**Comment by yuanqing** [comment]
This issue is fixed as of `1.4.0`
